src/templates/default_nativeapp/.gitignore

```diff
@@ -0,0 +1 @@
+/snowflake.*.yml
```

Comment:
I think we should specifically gitignore just the local one.

Reply:
I'm good with that, want to voice one thing: if user decides to name their file `snowflake.debug.yml` instead of local, then they might end up committing that. 
But I also take your point that we want to be able to commit override files that target specific connections. In which case, having those files is more important than having accidentally checked in a local/debug file, which can always be removed.

Reply:
Let's revisit after MVP. I have a feeling we'll likely be hardcoding the name `snowflake.local.yml`, and when we introduce e.g. an `-e` command line switch we can start to talk about what names make sense here.